src/client/pages/Offer/Introduction/Sidebar/index.tsx

```diff
@@ -176,7 +176,7 @@ export const Sidebar: React.FC<SidebarProps> = ({
     }
   }, [quoteCartId, removeCampaignCode])
 
-  const showRemoveCampaignButton = campaign != null
+  const showRemoveCampaignButton = campaign?.__typename === 'Campaign'
```

Comment:
What was the reason for this change? Could it also be wrapped in a utility function?

Reply:
Same question here.

Reply:
The original code assumed campaign could be null, now it can only be there or be undefined (updated in the props). 

As you may have seen in the selectors I have created, I have started linking the type check as a better way to know if something exists. But I will change this to a `campaign !== undefined`

Reply:
Fixed